app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, html: { class: 'd-none' }, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
```

Comment:
- [ ] As you can see in the Screencast below, I observed that the sign-up form is not visible on `http://localhost:3000/users/sign_up`. It appears that the signup form has been assigned the `d-none` class, which is causing it to be hidden. It would be greatly appreciated if you could resolve this issue.

  [Screencast from 09-08-2023 10:39:04 PM.webm](https://github.com/benja27/my-first-blog-with-rails/assets/92303891/2178eeec-587a-4d04-950f-a8e08a8ae0f9)

Reply:
done!